src/Orleans.Core/Lifecycle/LifecycleSubject.cs

```diff
@@ -47,7 +47,8 @@ public LifecycleSubject(ILogger<LifecycleSubject> logger)
                     Stopwatch stopWatch = Stopwatch.StartNew();
                     await Task.WhenAll(observerGroup.Select(orderedObserver => WrapExecution(ct, orderedObserver.Observer.OnStart)));
                     stopWatch.Stop();
-                    this.logger?.Info(ErrorCode.SiloStartPerfMeasure, $"Starting lifecycle stage {this.highStage} took {stopWatch.ElapsedMilliseconds} Milliseconds");
+                    if(this.logger.IsEnabled(LogLevel.Trace))
+                        this.logger?.Trace(ErrorCode.SiloStartPerfMeasure, $"Starting lifecycle stage {this.highStage} took {stopWatch.ElapsedMilliseconds} Milliseconds");
```

Comment:
The desired behavior is for silo and cluster client to report this as info, and grains to report this as trace.

In general, LifecycleSubject is a generic base class for most lifecycles.  If logging need be specialized, we should either not having it in the base class or provide specializations to override the behavior as needed.

Reply:
will go think of a way to do that